### ISSUE TITLE
StackMob::Event: u.a.s.l.EquipListener (EntityPickupItemEvent) causing performance hit

### ISSUE THREAD

**Issue by cloakfox** [Nathat23/StackMob-5#182]
https://timings.aikar.co/?id=c3c2b0d160f84b63a63a9284aa2c0765

Using version 5.4.1

**Comment by Nathat23** [comment]
Setting 'events.equip.enabled' to 'false' would fix it, I'll add some extra checks to try a reduce the performance impact. But generally it seems there's a lot of items being dropped?

**Comment by cloakfox** [comment]
Disabling the equip check didnt improve anything at all (with plugin reload)

I've just disabled stackmob as it's not improving performance for us at all.

**Comment by Nathat23** [comment]
with '/sm reload'? you'd have to restart the server for it to take effect.

**Comment by cloakfox** [comment]
Yes I used /stackmob reload.

Im no longer using a stacker plugin, they cause more lag then they prevent.

**Comment by Aurelien30000** [comment]
> Yes I used /stackmob reload.
> 
> Im no longer using a stacker plugin, they cause more lag then they prevent.

Right, they cause lag, however they prevent more lag if correctly used and in a judicious way, with config options that could cause issues for example ^^
